docs/usage/help.rst.inc

```diff
@@ -18,12 +18,12 @@ unwanted ones. Patterns can be used
 - for ``PATH`` arguments that explicitly support them.
 
 Borg always stores all file paths normalized and relative to the
-current recursion root. The recursion root is also named ``PATH`` in
-Borg commands like `borg create` that do a file discovery, so do not
-confuse the root with the ``PATH`` argument of e.g. `borg extract`.
+first directory of the current recursion root. The recursion root is also named
+``PATH`` in Borg commands like `borg create` that do a file discovery, so do
+not confuse the root with the ``PATH`` argument of e.g. `borg extract`.
```

Comment:
I find the sentence involving `PATH` confusing.  Why say that because the recursion root is named `PATH` in some commands, you should *not* confuse a `PATH` argument with a recursion root?  Should it be more like "While the recursion root is named PATH in Borg commands like `borg create` that do file discovery, in other commands like `borg extract`, the `PATH` argument should be a normalized path."  ?

Reply:
Yeah, for create `PATH` argument is a recursion root, for extract it is for matching what to extract.